src/Appium.Net/Appium/Service/AppiumCommandExecutor.cs

```diff
@@ -54,13 +55,19 @@ internal AppiumCommandExecutor(AppiumLocalService service, TimeSpan timeForTheSe
         }
 
         public Response Execute(Command commandToExecute)
+        {
+            return Task.Run(() => RealExecutor.ExecuteAsync(commandToExecute)).GetAwaiter().GetResult();
+        }
+
+        public async Task<Response> ExecuteAsync(Command commandToExecute)
         {
             Response result = null;
 
             try
             {
                 bool newSession = HandleNewSessionCommand(commandToExecute);
-                result = RealExecutor.Execute(commandToExecute);
+                result = Task.Run(() => RealExecutor.ExecuteAsync(commandToExecute)).GetAwaiter().GetResult();
```

Comment:
`ConfigureAwait(false)` is generally more efficient because it does not incur the overhead of creating a new thread, and it avoids potential deadlocks or unnecessary blocking caused by capturing the current synchronization context.
You can read more about it in this blog: https://blog.stephencleary.com/2012/02/async-and-await.html